project.clj

```diff
@@ -1,40 +1,45 @@
-(defn- with-ns [n] (str "org.broadinstitute.firecloud-ui." n))
-
 (defproject org.broadinstitute/firecloud-ui "0.0.1"
   :dependencies
   [
-   [dmohs/react "0.2.11"]
-   [org.clojure/clojure "1.7.0"]
-   [org.clojure/clojurescript "1.7.228"]
+   [dmohs/react "1.0.2+15.0.2"]
+   [org.clojure/clojure "1.8.0"]
+   [org.clojure/clojurescript "1.9.293"]
```

Comment:
That's the only one I know about. However, I wouldn't trust it for anything besides a trivial expression, since the versions will always be different than our own code. Instead, I'd recommend the REPL started by Figwheel, which is running our versions.